runtime/common/src/constants.rs

```diff
@@ -107,15 +107,20 @@ pub type FrameSystemMaxConsumers = ConstU32<16>;
 // -end- Frame System Pallet ---
 
 // --- MSA Pallet ---
+// The maximum number of public keys per MSA
```

Comment:
Just in this file, let's make these all `///` comments so they are made part of the rust docs (I just verified that they're included in the common_runtime generated documentation if so).  The others are already documented in their own pallets. 

IntelliJ's hover-help (and maybe VSCode's too) also will pick the comments up that way, and it's handy to have so you don't have to open the file to look.

Reply:
Great suggestion, Shannon!